vision/pipeline.py

```diff
@@ -77,8 +81,7 @@ def run(self, prev_state):
             pass  # raise AttributeError(f"Unrecognized state: {state}")
 
         ##
-        for bbox in bboxes:
-            self.vision_communication.put(bbox, self.PUT_TIMEOUT)
+        self.vision_communication.put((self.time, bboxes), self.PUT_TIMEOUT)
```

Comment:
Just send the current time in datetime format, then I can prune old ones

Reply:
Is there any specific function that would work best?

Reply:
datetime.datetime() https://docs.python.org/3/library/datetime.html

Reply:
datetime.datetime.now() it is